matrix/common/etl/__init__.py

```diff
@@ -76,7 +76,10 @@ def transform_bundle(bundle_uuid: str, bundle_version: str, bundle_path: str, ex
     ]
 
     for transformer in transformers:
-        transformer.transform(bundle_path)
+        try:
+            transformer.transform(bundle_path)
+        except Exception as e:
+            logger.error(f"Failed to transform bundle {bundle_uuid}.{bundle_version}.", e)
```

Comment:
logging an error doesn't seem sufficient

Reply:
Agreed. As we discussed, errors should be propagated and stored in a DLQ. Work captured [here](https://app.zenhub.com/workspaces/dcp-backlogs-5ac7bcf9465cb172b77760d9/issues/humancellatlas/matrix-service/227)